etl/steps/data/garden/energy/2023-12-12/energy_mix.meta.yml

```diff
@@ -1,25 +1,1266 @@
 definitions:
   common:
+    processing_level: major
     presentation:
       topic_tags:
-      - Energy mix
+      - "Energy"
```

Comment:
I don't know if it matters that the topic here has quotes? I just noticed it as being unusual